### PR TITLE
Added Objectives to css.md

### PR DESCRIPTION
**Pull request by pydevsg**
Added objectives Changing and Removing Styles from CSS using Chrome Dev Tools .
Fixed Issue #698 .

### CODE REVIEW

File: css/css.md [hunk 1]
```diff
@@ -38,6 +38,8 @@ About 4.5 hours
 - Describe the hierarchy of specificity for CSS selectors
 - Describe the different parts of the "Box" model
 - Effectively use the `display` property
+- Changing styles from CSS using `Chrome Dev Tools`
```
Comment (alodahl) [inline css/css.md]: can you state this in one line please? ie "Changing and removing..."

**Comment by alodahl** [comment]
@Kundan28 are you approving changes to the slides too?

File: css/css.md (hunk 1 above)
Comment (alodahl) [reply css/css.md]: also, can you format Chrome Dev Tools in another way?  This way denotes code or commands.

Comment (alodahl) [reply css/css.md]: still waiting on some small changes @pydevsg

Comment (pydevsg) [reply css/css.md]: I will make the changes as soon as possible @alodahl .

**Comment by alodahl** [comment]
Hi @pydevsg , are you still working on this issue?

**Comment by pydevsg** [comment]
I have made all the changes as said by you and @Kundan28  , waiting for the PR to be merged.

File: css/css.md (hunk 1 above)
Comment (alodahl) [reply css/css.md]: Did you change this one?

**Comment by alodahl** [comment]
@pydevsg , do you have 2 PRs out for issue #698?

**Comment by alodahl** [comment]
#737 #789

**Comment by pydevsg** [comment]
Yes, I have 2 PRS for issue #698 , I am closing this one working on #789

**Pull request closed without merging**